### PR TITLE
UP-2021 Using binary simple binary verification instead of hex

### PR DESCRIPTION
**Pull request by Carlos-Augusto**
Fixing niomon-decoder to identify msgpack from binary data arrary instead of encoding to hex.

### CODE REVIEW

File: src/main/scala/com/ubirch/decoding/Verify.scala [hunk 1]
```diff
@@ -42,34 +23,41 @@ class DefaultVerify(verifier: MultiKeyProtocolVerifier) extends Verify with Lazy
...
+      //Todo: Use cached KeyServiceClient
+      verifier
+        .verifyMulti(hardwareId, dataToVerify, 0, dataToVerify.length, signature) match {
 
+        case Some(key) =>
+          logger.info(s"signature_verified_for=$hardwareId", v("requestId", requestId))
+          record.withExtraHeaders(("algorithm", key.getSignatureAlgorithm))
         case None =>
-          val errorMsg = s"Header with key $HARDWARE_ID_HEADER_KEY is missing. Cannot verify msgPack."
+          val errorMsg = s"signature verification failed for msgPack of hardwareId $hardwareId."
```
Comment (thinkberg) [inline src/main/scala/com/ubirch/decoding/Verify.scala:55]: I think the short name for msgpack is msgpack not msgPack

Comment (Carlos-Augusto) [reply src/main/scala/com/ubirch/decoding/Verify.scala:55]: Yes, msgpack is the correct short name

Comment (Carlos-Augusto) [reply src/main/scala/com/ubirch/decoding/Verify.scala:55]: Will do the corresponding changes

Comment (Carlos-Augusto) [reply src/main/scala/com/ubirch/decoding/Verify.scala:55]: It was resolved.

**Pull request merged**